beetsplug/convert.py

```diff
@@ -532,11 +534,15 @@ def convert_on_import(self, lib, item):
 
             # Change the newly-imported database entry to point to the
             # converted file.
+            source_path = item.path
             item.path = dest
             item.write()
             item.read()  # Load new audio information data.
             item.store()
 
+            if self.config['delete_originals']:
+                util.remove(syspath(source_path), False)
```

Comment:
this is a tiny nitpick, but the `syspath` call here isn't necessary since it's called internally with `util.remove`

Reply:
That was part of the original PR, and to be honest I didn't really look too closely at it. I'll fix this. 👍🏻